templates/fragments/_email_alert.erb

```diff
@@ -1,4 +1,27 @@
-<email_alerts>
-  <email_to><%= @alert_email %></email_to>
-  <% if @alert_group != false -%><group><% @alert_group.each do |gr| -%><%= gr %>,<% end %></group><% end %>
-</email_alerts>
+  <email_alerts>
+    <email_to><%= @alert_email %></email_to>
+    <%- if @level -%>
+    <level><%= @level %></level>
+    <%- end -%>
+    <%- if @alert_group -%>
+    <group><%= [@alert_group].flatten.compact.join(',') %></group>
```

Comment:
Rule groups for [group](https://documentation.wazuh.com/4.0/user-manual/reference/ossec-conf/email_alerts.html#group) options should be separated with a pipe character (“|”).

Reply:
Hi @Zenidd,

please review changes.

Thank you!